src/main/java/org/folio/circulation/resources/ItemsInTransitResource.java

```diff
@@ -115,8 +115,12 @@ private void getMany(RoutingContext routingContext) {
     return CompletableFuture.completedFuture(Result.succeeded(item))
       .thenComposeAsync(itemRepository::fetchItemRelatedRecords)
       .thenComposeAsync(result -> result
-        .combineAfter(it -> servicePointRepository
-          .getServicePointById(it.getInTransitDestinationServicePointId()), Item::updateDestinationServicePoint));
+        .combineAfter(currentItem -> servicePointRepository
+          .getServicePointById(currentItem.getInTransitDestinationServicePointId()), Item::updateDestinationServicePoint))
```

Comment:
I imagine `it` was used as a shorter name to help with line length. Does this extend over the maximum line length?

Reply:
Split into 2 lines. It seems to me `currentItem ` is a more readable name.

```